### ISSUE TITLE
InnerBlocks multiple templates - using TAB (keyboard) goes into sidebar settings instead of next block

### ISSUE THREAD

**Issue by ivkatic** [WordPress/gutenberg#28613]
## Description
If you have InnerBlocks setup with multiple blocks the tabbing (using keyboard TAB) is not going through all the fields inside this main block, but rather goes to the right sidebar. See here https://www.dropbox.com/s/7m9dvgx0luabjb6/tabbing.gif?dl=0 .
I'm not sure if this works as intended or it's a bug, but if this is as intended is there a way to make this tab through main block only?

## Step-by-step reproduction instructions
1. Custom block with InnerBlocks multiple templates
2. Try using TAB to go through all fields and it "jumps" to the right sidebar

## Expected behaviour
Tab through all fields inside main block

## Screenshots or screen recording (optional)
https://www.dropbox.com/s/7m9dvgx0luabjb6/tabbing.gif?dl=0

## WordPress information
- WordPress version: "5.6.0"
- Gutenberg version: latest that comes with WP 5.6 I guess
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? Yes

## Device information
- Device: "Desktop"
- Operating system: Any 
- Browser: Any

**Comment by tellthemachines** [comment]
When a block is selected, pressing the Tab key takes us to the sidebar; this happens whether the selected block has inner blocks or not. That is the intended behaviour, but there are a few different opinions on whether it should be changed/improved. 

I'm going to close this issue as a duplicate of #24728, because there's extensive discussion of this behaviour over there. In any case, thanks for the report @ivkatic !
